javascript/multiBracketValidation/README.md

```diff
@@ -0,0 +1,23 @@
+# Code Challenge: Multi Bracket Validation
+
+## CC=Code Challenge/Pair Programming
+
+Tina Myers
+
+## Languages:
+
+javascript
+
+## Approach/Efficiency:
+
```

Comment:
I would suggest adding some information here about how you went about solving this